core/src/test/java/com/google/common/truth/IterableSubjectTest.java

```diff
@@ -388,8 +388,10 @@ public void iterableContainsAtLeastInOrderWithFailure() {
     assertFailureKeys(
         "required elements were all found, but order was wrong",
         "expected order for required elements",
+        "actual order for required elements",
```

Comment:
The error message after this change would be:

```
required elements were all found, but order was wrong
expected order for required elements: [<expected order>]
actual order for required elements  : [<actual order>]
but was                             : [<full list>>]
```

I don't think that makes sense grammatically.